README.md

```diff
@@ -29,23 +29,153 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
+  * Example:
+     * `const d = new DurationFormat();`
+     * `d.format(new Temporal.Duration.from({hours: 3, minutes: 4});`
+
+
+### Formatting width
+Users want to determine several types of the formatting width as following
+  | Format width  | Example               |
+  | -----------   | -----------           |
+  |  Wide         |1 hour and 50 minutes  |
+  |  Short        |1 hr, 50 min           |
+  |  Narrow       |1h 50 m                |
+  |  Dotted       |1: 50: 00              |
+
+
+#### Design
+  * The user can determine the formatting width using a  parameter `style` and the value of this parameter take a ` string` as following:
+      * `“wide”`
+      * `“short”`
+      * `“narrow”`
+      * `“dotted”`
+
+### Supported Time Fields
+Users need the following fields to be supported
+  * Year
+  * Month
+  * Week
+  * Day
+  * Hour
+  * Minute
+  * Second
+  * Millisecond
+  * Microsecond
+  * Nano Second
+
+#### Design
+We are going to support the following fields as a beginning
+  * Week
+  * Day
+  * Hour
+  * Minute
+  * Second
+  * Millisecond
+  * Microsecond
+  * Nano Second
+
+### Determining Time Fields
+Users need to determine the time fields,
+ * For example: 6059 seconds
+   * Hours, Minutes, Seconds
+     * 1 hours, 40 minutes and  59 seconds
+   * Minutes and Seconds
+     * 100 minutes and 59 seconds.
+ 
+#### Design
+   * Users can determine the time fields using `LargestField` and `SmallestField` parameters.
+   * Example: 
+   ```javascript
+    {
+      LargestField: "hour",
+      SmallestField: "second"
+    }
+  ```
+   * Default values
+    ```javascript
+         LargestField: "Day",
+         SmallestField: "second"
+    ```
+### Hide zero-value fields
+- Users needs to hide zero-value fields depends in some criteria
+  * Hide all zero-value fields
+    * Hours, Minutes, Seconds (without the `hide` option)
+      * 1 hours, 0 minutes and 59 seconds.
+    * Hours, Minutes, Seconds (with the `hide` option)
+      * 1 hours and 59 seconds.
+  * Hide-upper-limit with zero-valued.
+    * Hours, Minutes, Seconds, MilliSeconds (without the `hide-upper-limit` option)
+      * 0 hr, 20 min, 0 sec, 100 msec .
+    * Hours, Minutes, Seconds MilliSeconds(with the `hide-upper-limit` option)
+      * 20 min, 0 sec, 100 msec.
+  * Hide-lower-limit with zero-valued.
+
+#### Design
+- Users can set the `HidingZeroValuedOption` parameter  by one of the following values:
+    * `"ALL"` // Hide all the fields that have a zero-valued.
+    * `“LEADING_AND_TRAILING”` // Hide all the zero fields in the leading or the training
+    * `“LEADING_ONLY”`
+    * `“TRAILING_ONLY”` 
+    * `“NONE”` // Do not hide any zero-valued fields.
+
+- Default Value: `“NONE”`
+
+### Round 
+  - Users wants to decide if they are going to round the smallest field or not.
+  - for example: 
+    * Without rounding option
+      * 1 hour and 30.6 minutes.
+    * With rounding option
+      * 1 hour and 31 minutes.
```

Comment:
Rounding is also something that's being discussed in `Temporal`. My personal opinion is to leave this out from the current draft atleast, but anyway this should be completely dependent on the decision on balancing, since I consider that the same rules would apply here.

Reply:
I agree with you, but I would like to discuss this next week when we are going to decide where the arithmetic should happened